pantheon.php

```diff
@@ -3,13 +3,15 @@
  * Plugin Name: Pantheon
  * Plugin URI: https://pantheon.io/
  * Description: Building on Pantheon's and WordPress's strengths, together.
- * Version: 1.0.0
+ * Version: 1.2.0
  * Author: Pantheon
  * Author URI: https://pantheon.io/
  *
  * @package pantheon
  */
 
+define( 'PANTHEON_MU_PLUGIN_VERSION', '1.2.0' );
```

Comment:
- I would put this in README as well
- Why 1.2 and not 1.1 (I know this should have been incremented many times along the way)

Reply:
I consider, since WPCS -> 3.0 is a big deal and we weren't linting at all previously, this to be a larger than patch update.

Reply:
@pwtyler you mean list the version number in the readme? that would be _one more thing_ we need to update when we cut new releases.